### ISSUE TITLE
Contributing to auto-generated files?

### ISSUE THREAD

**Issue by explody** [okta/okta-sdk-python#243]
I was just about to work on #230 when I noticed the warning at the top of the file:

```
# AUTO-GENERATED! DO NOT EDIT FILE DIRECTLY
# SEE CONTRIBUTOR DOCUMENTATION
```

I checked:  https://github.com/okta/okta-sdk-python/blob/master/CONTRIBUTING.md

But there's nothing there on the subject.  I'm guessing some or all of the models are generated from OpenAPI or similar?  The fix for Group Profiles is pretty simple, but does this mean that we'd need to wait for the OpenAPI data to update instead of adding profile support directly?

**Comment by serhiibuniak-okta** [comment]
@explody yes, all models are being generated in accordance with openapi spec: https://github.com/okta/okta-management-openapi-spec
And for given particular case I've provided PR - https://github.com/okta/okta-sdk-python/pull/238 (though, we haven't released it yet). I think, we should update contributing guide to explain this nuance, I'll put such task to our backlog.
Does it work for you?

**Comment by explody** [comment]
Looks great, pretty much exactly what I was about to try manually :P 

I suppose the question is for cases like this - is it a matter of keeping model code up to date with the changing OpenAPI spec?  Or does this involve updating the spec itself first?

Put another way, is this something 3rd party contributors can do, or should we just expect to wait on the models to catch up to the spec?

Either way is fine, just for future reference...

**Comment by serhiibuniak-okta** [comment]
Yes, we need to update spec first. We can't release sdk if it diverges from spec. On the other hand, 3rd party contributors can generate sdk based on a new spec release and create a PR (if we didn't do this in time, of course).

**Comment by explody** [comment]
Got it, thank you.  Just FYI PR #238 get close but I don't see group_profile.py being updated yet, so the profile data still doesn't make it to the client (just confirmed this locally).  But, we can take this conversation back to #230 

Thanks!
